setup.py

```diff
@@ -16,6 +16,7 @@
     package_data={'': ['LICENSE']},
     include_package_data=True,
     install_requires=[
+        'functools32',
```

Comment:
looks like `functools32` installs only in py3..

`repoze.lru` / `from repoze.lru import lru_cache` works in all python versions